### ISSUE TITLE
Zone JSON is assumed to have 'template' key but that is not always the case

### ISSUE THREAD

**Issue by jeroenvermeulen** [sandwave-io/realtimeregister-php#72]
This code:
https://github.com/sandwave-io/realtimeregister-php/blob/475e386887600382940243dc22be91bf6f2b0b97/src/Domain/Zone.php#L17-L23
barfs on this data from RTR Live:
```
{
  "domainName": "somedomain.nl",
  "registrant": "some_registrant",
  "status": [
    "OK"
  ],
  "authcode": "___secret___",
  "createdDate": "2018-08-03T09:49:40Z",
  "updatedDate": "2023-04-19T11:46:06Z",
  "expiryDate": "2023-11-03T09:49:40Z",
  "ns": [
    "ns1.yoursrs.com",
    "ns2.yoursrs.com"
  ],
  "autoRenew": true,
  "autoRenewPeriod": 12,
  "zone": {
    "id": 1334243663,
    "service": "BASIC",
    "dnssec": false
  },
  "premium": false,
  "gateway": false,
  "privacyProtect": false,
  "contacts": [
    {
      "role": "ADMIN",
      "handle": "some_registrant"
    },
    {
      "role": "TECH",
      "handle": "some_registrant"
    },
    {
      "role": "BILLING",
      "handle": "some_registrant"
    }
  ],
  "customer": "savvii",
  "registry": "Sidn"
}
```
Error:
```
PHP Warning:  Undefined array key "template" in vendor/sandwave-io/realtimeregister-php/src/Domain/Zone.php on line 20
```

**Comment by websmurf** [comment]
I've created a pull request to fix this

**Comment by jeroenvermeulen** [comment]
Wow that is quick! Thank you, looks good as far as I can see.

**Comment by A1rPun** [comment]
Fixed in #73
